test/e2e/apimachinery/resource_quota.go

```diff
@@ -1418,7 +1418,7 @@ func waitForResourceQuota(c clientset.Interface, ns, quotaName string, used v1.R
 		}
 		// verify that the quota shows the expected used resource values
 		for k, v := range used {
-			if actualValue, found := resourceQuota.Status.Used[k]; !found || (actualValue.Cmp(v) != 0) {
+			if actualValue, found := resourceQuota.Status.Used[k]; !found || (actualValue.Cmp(v) < 0) {
```

Comment:
Out of curiosity, why is the original condition incorrect? On the surface of things, it seems reasonable to want to wait for our actual used resource quota to match our expected used resource quota...